Source/Relativity.Test.Helpers/Relativity.Test.Helpers/TestHelper.cs

```diff
@@ -104,23 +104,18 @@ public Guid GetGuid(int workspaceID, int artifactID)
 		{
 			var keplerHelper = new KeplerHelper();
 
-			if (keplerHelper.ForceDbContext())
-			{
-				return GetGuidFromDbContext(workspaceID, artifactID);
-			}
-
+			if (keplerHelper.ForceDbContext()) return GetGuidFromDbContext(workspaceID, artifactID);

```

Comment:
Our usual standard (coming from Chandra's usual notes) is that we return at the end and not in the middle of a function.